src/main/java/org/breskul/model/SettingsForSession.java

```diff
@@ -0,0 +1,28 @@
+package org.breskul.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class SettingsForSession {
+
+    private boolean showSql = true;
+    private boolean enableDirtyChecker = true;
+
+
+    public static SettingsForSession createSettingsWithOffDirtyChecker() {
```

Comment:
I would suggest to use builder instead of this long method names. (If default and  + 1 constructor with two parameters are not enough)